store/src/main/java/org/apache/rocketmq/store/ConsumeQueue.java

```diff
@@ -774,11 +774,16 @@ private void doDispatchLmqQueue(DispatchRequest request, int maxRetries, String
         int queueId) {
         ConsumeQueueInterface cq = this.messageStore.findConsumeQueue(queueName, queueId);
         boolean canWrite = this.messageStore.getRunningFlags().isCQWriteable();
+        final long beginTimeMillis = this.messageStore.now();
         for (int i = 0; i < maxRetries && canWrite; i++) {
             boolean result = ((ConsumeQueue) cq).putMessagePositionInfo(request.getCommitLogOffset(), request.getMsgSize(),
                 request.getTagsCode(),
                 queueOffset);
             if (result) {
+                this.messageStore.getBrokerStatsManager().incTopicPutNums(queueName, 1, 1);
+                this.messageStore.getBrokerStatsManager().incTopicPutSize(queueName, request.getMsgSize());
+                this.messageStore.getBrokerStatsManager().incBrokerPutNums(queueName, 1);
```

Comment:
IMO, dispatch lmq just create a consume index, it didn't really put a message to broker, incBrokerPutNums here maybe inappropriate.

Reply:
agree